src/ring_theory/unique_factorization_domain.lean

```diff
@@ -943,6 +943,18 @@ begin
   { exact count_normalized_factors_eq hp hnorm hle hlt }
 end
 
+lemma max_power_factor {a₀ : R} {x : R} (h : a₀ ≠ 0) (hx : irreducible x) [nontrivial R] :
```

Comment:
One more lint fix, and we're good to go! If you're happy with this fix, you can `bors merge` the PR, finally :)
```suggestion
lemma max_power_factor {a₀ : R} {x : R} (h : a₀ ≠ 0) (hx : irreducible x) :
```